packages/xo-server/src/api/backup-ng.js

```diff
@@ -138,6 +142,13 @@ runJob.params = {
   schedule: {
     type: 'string',
   },
+  settings: {
+    type: 'object',
+    properties: {
+      '*': { type: 'object' },
```

Comment:
`'*'` -> `''`

Reply:
[`*`](https://github.com/Atinux/schema-inspector#v_properties) is a special property which means any property.